packages/fxa-content-server/server/lib/amplitude.js

```diff
@@ -130,15 +130,29 @@ const EVENTS = {
   },
 
   // Reset password confirmation
-  'screen.settings.reset-password-confirmed': {
+  'screen.reset-password-confirmed': {
     group: GROUPS.settings,
```

Comment:
This is from your previous PR right? We also need to change the `group` of these, I'm fine with you doing that here or in a 1-pointer follow up to look at these events / add them to the spreadsheet.

Reply:
I'll change the group now! I'm updating Ready/index.tsx in the next Storybook PR If that works for you!